uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/AdbShellUtils.java

```diff
@@ -38,7 +41,10 @@
  */
 public class AdbShellUtils {
 
+    private final static String TAG = AdbShellUtils.class.getSimpleName();
+
     public static String executeShellCommand(@NonNull final String command) {
+        Logger.i(TAG, "Execute Shell Command..");
```

Comment:
let's log the command in the log message

Reply:
added.